packages/ui/src/components/FlipCard.vue

```diff
@@ -5,14 +5,27 @@
       @click="showStats = !showStats"
       :class="{ flipped: showStats }"
     >
-      <div class="face front" :class="{ horizontal }">
-        <BufficornImage :name="name" />
-        <PolarChart :stats="normalizedData" />
+      <div
+        class="face front"
+        :class="{ horizontal, background: backgroundFront, selected }"
+      >
+        <BufficornImage :class="{ selected }" :name="name" />
```

Comment:
This image shouldn't be visible when back is active

```suggestion
        <BufficornImage :class="{ selected, hidden: !!backBack }" :name="name" />
        ...
        
        <style>
        .hidden {
          visibility: hidden;
        }
        </style
```